cmake/vcpkg.json

```diff
@@ -30,15 +30,15 @@
     },
     "ms-gsl",
     "nlohmann-json",
-    {
-      "name": "nsync",
-      "platform": "!windows"
-    },
     {
       "name": "nsync",
```

Comment:
@luncliff  , we no longer use nsync. This part can be removed.

Reply:
Yes. I saw https://github.com/microsoft/onnxruntime/pull/20413
I will remove the others before working in WebAssembly build
 https://github.com/microsoft/onnxruntime/blob/4134cd9e42c412e5770d84137aad5f7161ca5515/cmake/onnxruntime_webassembly.cmake#L229